src/components/OfflineIndicator.tsx

```diff
@@ -46,7 +48,25 @@ function OfflineIndicator({style, containerStyles}: OfflineIndicatorProps) {
                 width={variables.iconSizeSmall}
                 height={variables.iconSizeSmall}
             />
-            <Text style={[styles.ml3, styles.chatItemComposeSecondaryRowSubText]}>{translate('common.youAppearToBeOffline')}</Text>
+            <Text style={[styles.ml3, styles.chatItemComposeSecondaryRowSubText]}>
+                {
+                    // Do not reorder the condition, we only show unreachability message when online and backend is unreachable
```

Comment:
I'm wondering if we can change the code and/or comments so it's clear what our intention is and nobody is "tempted" to change the code into a wrong direction, but without using "orders" like "Do not reorder...".